mappings/net/minecraft/client/render/RenderLayers.mapping

```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/class_4696 net/minecraft/client/render/RenderLayers
 	FIELD field_21472 fancyGraphicsOrBetter Z
 	METHOD method_23678 getItemLayer (Lnet/minecraft/class_1799;Z)Lnet/minecraft/class_1921;
 		ARG 0 stack
+		ARG 1 direct
```

Comment:
This might be a scope creep on the PR, but it would be nice if you could document the `direct` parameter and explain what it does like the PR description which was pretty clear.

You don't need to do it to every render layer as we can get those other ones in later PRs.